src/macaron/slsa_analyzer/checks/detect_malicious_metadata_check.py

```diff
@@ -40,13 +42,16 @@ class MaliciousMetadataFacts(CheckFacts):
     #: The primary key.
     id: Mapped[int] = mapped_column(ForeignKey("_check_facts.id"), primary_key=True)  # noqa: A003
 
+    #: Known malware.
+    known_malware: Mapped[str | None] = mapped_column(
+        String, nullable=True, info={"justification": JustificationType.HREF}
+    )
+
     #: Detailed information about the analysis.
     detail_information: Mapped[dict[str, JsonType]] = mapped_column(DBJsonDict, nullable=False)
 
-    #: The result of analysis, which is of dict[Heuristics, HeuristicResult] type.
-    result: Mapped[dict[Heuristics, HeuristicResult]] = mapped_column(
-        DBJsonDict, nullable=False, info={"justification": JustificationType.TEXT}
-    )
+    #: The result of analysis, which can be an empty dictionary.
+    result: Mapped[dict] = mapped_column(DBJsonDict, nullable=False, info={"justification": JustificationType.TEXT})
```

Comment:
Why change `dict[Heuristics, HeuristicResult]]` to `dict`? In the new case it is empty, but that is still consistent with the old type.

Reply:
That's right. I don't need to change `dict[Heuristics, HeuristicResult]]`.

Reply:
See commit [a61bb94](https://github.com/oracle/macaron/pull/922/commits/a61bb94fb8403268e0931445dc256b641f35e4d2).